src/parse.jl

```diff
@@ -1,6 +1,13 @@
 import Compat.Dates: DateFormat, DatePart, tryparsenext, format, min_width, max_width,
     default_format
 
+# Handle Nullable deprecation on Julia 0.7
+if VERSION < v"0.7.0-DEV.3017"
+    nullable(T::Type, x) = Nullable{T}(x)
+else
+    nullable(T::Type, x) = x  # Ignore nullable container type T
+end
```

Comment:
I'm confused by this. Could you not use `Nullable` from the Nullables package, which is already a dependency, or switch to `Some`, which is available in Compat?

Reply:
When `Nullable` was removed from Base the parsing interface in Dates was also changed to no longer use `Nullable`. Specifically the `tryparsenext` interface now expects a `Tuple{Union{T, Nothing}, Int}` where previously a `Tuple{Nullable{T}, Int}`.

Since I am hooking into the code provided by Dates I need to provide exactly what that interface requires.

Reply:
Okay, thanks for the explanation.